faq_bot/bot/script/transitions.py

```diff
@@ -0,0 +1,31 @@
+"""
+Transitions
```

Comment:
Maybe we should remain this file to conditions.py and move qa_transitions to script.py for consistency with older scripts
```suggestion
Conditions
```

Reply:
I don't think we should do that. Then we'd have a file defining two functions that used to be one-liners.

I don't think it would be confusing for anyone familiar with older examples that transitions moved to another file.

Reply:
ok